main.js

```diff
@@ -0,0 +1,113 @@
+/* eslint-disable */
+
+let objectlist =[
+  {
+    bookname: 'Java books',
+    author: 'Ali',
+  },
+  {
+    bookname: 'CSS Books',
+    author: 'Ahmad',
+  },
+  {
+    bookname:'HTML Books',
+    author:'Ali'
+  }
+];
+
+function generateBooks({
+  bookname, author
+}){
+  return `
+  <li>
+  <span class="name">${bookname}</span><br>
+  <span class="name">${author}</span><br>
+  <button class="delete">delete </button><br>
+</li>
+  `;
+}
+
```

Comment:
- [ ] The books are stored in the local storage successfully but when I refresh the page I do not see books on the screen I added previously because there is no function that loads data from the local storage and displays it on the screen. Please make sure after refreshing the page users can see their previously added books on the display(That is why we use local storage so we can save our data and load them even if our application gets refreshed)